Omikron/Factfinder/Model/Export/Product.php

```diff
@@ -158,19 +163,35 @@ protected function buildFeedRow($product, $store)
             'ImageUrl',
             'Price',
             'Manufacturer',
-            'Attributes',
             'CategoryPath',
             'Availability',
             'EAN',
             'MagentoEntityId'
         ];
+
+        $additionalAttributes = $this->getAdditionalAttributes($store);
+        if (!empty($additionalAttributes)) {
+            $attributes = \array_unique(\array_merge($attributes, explode(',', $additionalAttributes)));
+        }
+
         foreach ($attributes as $attribute) {
             $row[$attribute] = $this->helperProduct->get($attribute, $product, $store);
         }
 
         return $row;
     }
 
+    /**
+     * Get the additional attribute fields for the store
+     *
+     * @param \Magento\Store\Api\Data\StoreInterface $store
+     * @return mixed
+     */
+    protected function getAdditionalAttributes($store)
```

Comment:
This function is duplicate from Omikron\Factfinder\Helper\Product.

Reply:
Yeah good point, I should just inject the Product Helper instead.